flink-end-to-end-tests/flink-tpcds-test/src/main/java/org/apache/flink/table/tpcds/TpcdsTestProgram.java

```diff
@@ -123,8 +123,9 @@ private static TableEnvironment prepareTableEnv(String sourceTablePath, Boolean
 		TableEnvironment tEnv = TableEnvironment.create(environmentSettings);
 
 		//config Optimizer parameters
-		tEnv.getConfig().getConfiguration()
-				.setString(ExecutionConfigOptions.TABLE_EXEC_SHUFFLE_MODE, ShuffleMode.BATCH.toString());
+		tEnv.getConfig().getConfiguration().setString(
```

Comment:
Consider default value is `ALL_EDGES_BLOCKING`, we can remove this code.

Reply:
Ok.